### ISSUE TITLE
make dist not working

### ISSUE THREAD

**Issue by michelezaccagnini** [Cycling74/rnbo.example.vcvrack#4]
I cloned the repo, did ```export RACK_SDK=<SDK dir>```, MSYS is installed (I have successfully created a VCV module).
I get this error
```Makefile:20: ../../plugin.mk: No such file or directory
make: *** No rule to make target '../../plugin.mk'.  Stop.
```

**Comment by michelezaccagnini** [comment]
I don't

**Comment by isabelgk** [comment]
Okay, in that case, try clearing out the `rnbo-export` directory and re-exporting with the name `rnbo_source.cpp`.

**Comment by isabelgk** [comment]
> it needs to have that specific name?

Yes, currently that's the case. I see it's not called out in the documentation anywhere, so that might need to be updated.

**Comment by michelezaccagnini** [comment]
Ok, it worked! Thanks so much. This is a great tool...

**Comment by isabelgk** [comment]
Wonderful! I'll update the README to clarify this current requirement.

**Comment by michelezaccagnini** [comment]
Good morning Isabel,

I created two issues on GitHub. The main one for me would be how to consolidate multiple modules in a VCV plugin, since they need to refer to same-named “rnbo-export” and “rnbo_source”. I know I have to change the json to add different SOURCES and rename things. I think this is related to the first issue I created.
I would like to submit this package to VCV library (it already exists as a package in Max), which brings me to the second issue: how to credit you and Cycling74 in this VCV release, whenever it happens.
Thanks for any suggestions!

Michele

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
